app/views/profiles/delete.html.erb

```diff
@@ -0,0 +1,38 @@
+<% @title = t('.title') %>
+
+<% unless @only_owner_gems.empty? %>
+  <div class="t-body">
+    <p class="page__subheading"><%= t '.list_only_owner' %>:</p>
+  </div>
+  <div id="profile">
+    <div class="profile-list">
+      <ul>
+        <%= render :partial => 'rubygem', collection: @only_owner_gems, locals: {graph: false, owners: true}  %>
```

Comment:
Mix of 1.8 style hash keys + json style - let's prefer the json style. Also, I think it can just be `<%= render 'rubygem', collection: ... %>`